integrations/sms/integration.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""SMS integration."""
+
+from __future__ import unicode_literals
+
+from integrationmanager.exceptions import IntegrationSendEventError
+from integrationmanager.error_messages import TEST_CONNECTION_REQUIRED
+from integrationmanager.integration_utils import BaseIntegration
+
+import twilio
+import twilio.rest
+
+MAX_SMS_LEN = 140
+
+class SMSIntegration(BaseIntegration):
+
+    def test_connection(self, data):
+        errors = {}
+        fields = ["from_phone", "to_phone", "twilio_account_sid", "twilio_auth_token"]
+        for field in fields:
+            if not data.get(field):
+                errors[field] = [TEST_CONNECTION_REQUIRED]
+        if len(errors) > 0:
+            return False, errors
+
+        from_phone = data.get("from_phone")
+        to_phone = data.get("to_phone")
+        twilio_account_sid = data.get("twilio_account_sid")
+        twilio_auth_token = data.get("twilio_auth_token")
+        additional_message_data = data.get("additional_message_data")
+
+        try:
+            client = twilio.rest.Client(twilio_account_sid, twilio_auth_token)
+            body = "Test Alert! Hello World!"
+            if additional_message_data:
+                body = body + " " + additional_message_data
+            client.messages.create(to=to_phone, from_=from_phone, body=body)
```

Comment:
Does this actually send an SMS? Is there a way to test the module without spending twilio budget?
Maybe test auth and check number syntax?

Reply:
I recently did this for demisto, just make a request to `client.accounts` to validate the credentials
when sending an SMS you'll get an error if something's wrong - it's not worth spending people's money on tests